cmd/state/internal/cmdtree/push.go

```diff
@@ -20,7 +20,10 @@ func newPushCommand(prime *primer.Values) *captain.Command {
 		[]*captain.Flag{},
 		[]*captain.Argument{},
 		func(ccmd *captain.Command, args []string) error {
-			return pushRunner.Run()
+			if fail := pushRunner.Run(); fail != nil {
+				return fail
+			}
+			return nil
```

Comment:
You won't need this after you have the `next()` call just return the error. The error handling in `main` accounts for runners that return failures.

Reply:
This is needed or a nil return from pushRunner.Run will be a typed nil and pollute/confuse inspection down the line.